client/hosting/sites/components/style.scss

```diff
@@ -1118,21 +1118,3 @@
 	}
 }
 
-// Styles need to ensure they do not bleed into other areas of calypso once loaded.
-// If we only apply to layout__primary, etc., we will have bleed going from the dashboard to site level calypso.
-.is-global-sidebar-visible.is-group-sites-dashboard,
-.is-global-sidebar-visible.is-group-sites {
-	/* See wp-calypso/client/layout/style.scss for the original style - this is removed on sites dashboard to enable pagination to display properly. */
-	.layout__primary .preview-hidden {
-		@include breakpoint-deprecated( ">960px" ) {
-			padding-bottom: 0 !important;
-		}
-	}
-
-	.layout__content {
-		@include breakpoint-deprecated( "<960px" ) {
-			padding: 70px 24px 24px calc(var(--sidebar-width-min) + 1px);
-		}
-	}
-}
```

Comment:
We recently added the above selectors for global sidebar and dashboard areas because these styles were bleeding into my-home and other areas of calypso after visiting the dashboard, affecting the layout in those areas negatively.

This extra specificity, however, has then overridden other styles that we have written for general sidebar behavior and global sidebar behavior.  These styles also no longer seem necessary as their cases are covered elsewhere. It seems preferable to remove these styles, as opposed to trying to pile on more styles and specificity elsewhere to override them (which also feels a bit like whack-a-mole after a brief attempt).